cli/src/semgrep/output.py

```diff
@@ -496,9 +496,9 @@ def output(
                 ):
                     missed_count_line = f"💎 Missed out on {unit_str(missed_rule_count, 'pro rule')} since you aren't logged in!"
                     learn_more_url = with_color(
-                        Colors.cyan, "https://sg.run/rules", underline=True
+                        Colors.cyan, "https://github.com/opengrep", underline=True
```

Comment:
We should probably put the full repo url to opengrep until other things are settled, ie, https://github.com/opengrep/opengrep.